### ISSUE TITLE
Structure Mech Macro is not working[BUG]

### ISSUE THREAD

**Issue by Azmahel** [Eranziel/foundryvtt-lancer#261]
**Describe the bug**
the Structure Token Macro on a mech or npc with missing structure does not work, putting an exception in chat instead.

**To Reproduce**
1.  set up a token with less than full structure
2. use the macro
3. check the console
4. See error: ```
foundry.js:3562 Uncaught (in promise) TypeError: str.replace is not a function
    at Localization.format (foundry.js:3562)
    at Object.localize (foundry.js:3743)
    at Object.c (handlebars.min.js:27)
    at Object.eval [as main] (eval at createFunctionContext (handlebars.min.js:29), <anonymous>:11:149)
    at c (handlebars.min.js:27)
    at d (handlebars.min.js:27)
    at e (handlebars.min.js:28)
    at renderTemplate (foundry.js:3627)
    at async Object.renderMacroTemplate (macros.ts:376)
```

**Expected behavior**
the structure macro should run and put its results into chat



**Platform (please complete the following information):**
 - OS: [ Windows]
 - Browser [firefox]
 - Foundry version [ 0.8.8]
 - Lancer system version [e.g. 0.9.5]

**Additional context**

Replacing the Template with the overheat template works fine, the only difference in these 2 templates is:
` <div class="lancer-stat-header clipped-top">{{localize "lancer.chat-card.title" title="STRUCTURE DAMAGE"}}</div>` ( Structure)
 `<div class="lancer-stat-header clipped-top">{{localize "lancer.chat-card.title.overheating"}}</div>` (Overheat)

**Comment by DMRaven** [comment]
I was also having issues with the structure macro, but not the same one listed above. Instead of an error, when trying to use the macro on a PC mech, I get no die roll in the chat window. Structure goes down by 1 and hit points are resumed to max. Negative hit points are no longer automatically handled like they were in Lancer v 0.1.2.

**Comment by Grygon** [comment]
Fixed in 0.9.6
